utils/callData/registrationCalls.ts

```diff
@@ -84,7 +84,7 @@ function mint(tokenId: number): Call {
   return {
     contractAddress: process.env.NEXT_PUBLIC_IDENTITY_CONTRACT as string,
     entrypoint: "mint",
-    calldata: [numberToString(tokenId)],
+    calldata: [numberToStringHex(tokenId)],
```

Comment:
Why are you changing here ?

Reply:
Paymaster returns a 500 error when we use decimal numbers, so I chose to use hexadecimals as they are supported by both wallet tx and paymaster